Zend/zend_builtin_functions.c

```diff
@@ -510,7 +510,7 @@ ZEND_FUNCTION(define)
 
 register_constant:
 	/* non persistent */
-	ZEND_CONSTANT_SET_FLAGS(&c, CONST_CS, PHP_USER_CONSTANT);
+    ZEND_CONSTANT_SET_FLAGS(&c, 0, PHP_USER_CONSTANT);
```

Comment:
Please use tabs instead of spaces for indentation of php-src C code:
```suggestion
	ZEND_CONSTANT_SET_FLAGS(&c, 0, PHP_USER_CONSTANT);
```